ipaserver/dnssec/ldapkeydb.py

```diff
@@ -414,7 +415,8 @@ def zone_keypairs(self):
     log = ipa_log_manager.root_logger
 
     # IPA framework initialization
-    ipalib.api.bootstrap(in_server=True, log=None)  # no logging to file
+    # no logging to file
+    ipalib.api.bootstrap(in_server=True, log=None, confdir=paths.ETC_IPA)
```

Comment:
NACK

Same reason as for ```ipa-compat-manage```

This is just some debugging code in ```if __name__ == '__main__'``` block. There is no harm to support IPA_CONFDIR